### DISCUSSION TITLE
UX: Web Search activation status more visible.

### DISCUSSION THREAD

**Discussion by papiche** [open-webui/open-webui#5834] [Ideas]
**Is your feature request related to a problem? Please describe.**
Sometimes, a prompt is needing external web search, but not for the next chat prompt
But as this feature is only visible clicking on "+" sign, I often forget it is set, so my chat prompt is diverted

**Describe the solution you'd like**
Move Wezb Search activation switch in a constant visible place. 

**Describe alternatives you've considered**
I can modify it after having forgotten its status

**Additional context**
![web-switch](https://github.com/user-attachments/assets/942800f8-57d0-4a33-821d-6111b1b0e183) overlay

**Comment by Genai-labs** [comment]
I completely agree with this suggestion. Having the Web Search activation switch in a more visible and constant location would be helpful. I also often forget that the feature is enabled, this can lead to unintended web searches, which interrupts the flow of conversation.

A more prominent and easily accessible switch would streamline the user experience, allowing us to quickly adjust the setting when needed without disrupting the chat. It would also reduce the chances of accidental searches when they aren't required.

This change would enhance usability and make interactions with the tool more intuitive.
